libfwupd/fwupd-release.c

```diff
@@ -1514,6 +1606,8 @@ fwupd_release_to_json (FwupdRelease *release, JsonBuilder *builder)
 		json_builder_end_array (builder);
 	}
 	fwupd_release_json_add_int (builder, FWUPD_RESULT_KEY_INSTALL_DURATION, priv->install_duration);
+	fwupd_release_json_add_string (builder, FWUPD_RESULT_KEY_DETACH_CAPTION, priv->detach_caption);
+	fwupd_release_json_add_string (builder, FWUPD_RESULT_KEY_DETACH_IMAGE, priv->detach_image);
```

Comment:
Do these need to show up in fwupdmgr/fwupdtool too?

Reply:
I wasn't sure; I erred on no, as we show the caption on the CLI when it's required

Reply:
Well 8bitdo plugin shows "internal" plugin errors.  Should these captions come back instead?

Reply:
They should do; see the first hunk in fu_engine_install()

Reply:
Ah, OK thanks!